docs/pages/repo/docs/getting-started/add-to-project.mdx

```diff
@@ -40,11 +40,13 @@ npm create vite@latest
   </Tab>
   <Tab>
     ```bash
-    pnpm install turbo --global
+    pnpm add turbo --global
```

Comment:
`install` should be correct here. (add is just `yarn`)

Reply:
It might work, but the `pnpm` docs suggest that [install](https://pnpm.io/cli/install) is for installing everything, whereas [add](https://pnpm.io/cli/add) is for adding a new dependency

Reply:
Ah TIL. That's a bit annoying since they mirror NPM in _almost_ everything else....